packages/wrangler/src/environment-variables/factory.ts

```diff
@@ -1,5 +1,56 @@
 import { UserError } from "../errors";
 
+/**
+ * Environment variables supported by Wrangler for configuration and authentication.
+ *
+ * ## Authentication & API Configuration
+ *
+ * - `CLOUDFLARE_ACCOUNT_ID` - Overrides the account ID for API requests. Can also be set in Wrangler config via `account_id` field.
```

Comment:
I think those should be on the variable themselves.

It is less readable like and will get out of sync quite quickly.

Reply:
Happy if you wish to put up another PR.